src/chains/index.ts

```diff
@@ -147,3 +147,4 @@ export { zkSyncTestnet } from './definitions/zkSyncTestnet.js'
 export { zetachainAthensTestnet } from './definitions/zetachainAthensTestnet.js'
 export { zora } from './definitions/zora.js'
 export { zoraTestnet } from './definitions/zoraTestnet.js'
+export { zoraSepolia } from './definitions/zoraSepolia.js'
```

Comment:
Can we please have these in alphabetical order? 
```suggestion
export { zoraSepolia } from './definitions/zoraSepolia.js'
export { zoraTestnet } from './definitions/zoraTestnet.js'
```